pragma-node/src/config.rs

```diff
@@ -3,20 +3,23 @@ use serde::Deserialize;
 use tokio::sync::OnceCell;
 
 #[derive(Debug, Deserialize)]
-struct ServerConfig {
-    host: String,
-    port: u16,
+pub struct ServerConfig {
+    pub host: String,
```

Comment:
let them private and create getters

Reply:
ah we no longer need those getters - leftover from when I added the onchain JsonRpcClient

removed the pub!